### DISCUSSION TITLE
When will a new version for the official Apple App Store be released?

### DISCUSSION THREAD

**Discussion by Visiraptor** [yattee/yattee#847] [Q&A]
Dear developer(s),

since the last stable version (1.5.1) was released almost a year ago, I wanted to ask when the next official release for a new App Store version will be? 
Or is there something that blocks/prevents it to happen (that is unknown to me right now)? 
Since I cannot (or maybe don't want to) go a "not so legal way" (e.g. like installing any alt store or ipa by hand) I am way more interessed in an official Apple App Store release that is newer than "1.5.1". There were a lot of "pre-releases" in the meanwhile with long changelogs (so in other words: "How many more 'pre-releases' needed to get a 'stable/latest' release on App Store?"). Hope there is something "Stable" planned in near future. Thank you for the good work. Wish you all a nice christmas and happy new year already.

PS: Almost all instances do not work for me anymore on 1.5.1. Hope that newer (beta or whatever) releases will make it work again.

**Comment by ghost** [comment]
I have also been wondering this. It is sad if there is never going to be another 'stable' release as there is seemingly no way to join testflight, and I don't really like 'beta' builds anyway.

It's doubly bothersome that like for you the current 'stable' doesn't work for me no matter what I do. So a new stable that actually played videos would be great!

**Comment by Visiraptor** [comment]
Dear Developer(s),

I wanted to follow up on my previous message from last December regarding a new stable release. I have not heard back, and I noticed that while there was a "Pre-release" last month (with a very long changelog), no new "Stable" version has been released. As it has been almost half a year since my initial inquiry, and considering that the latest stable release is over a year old, I kindly ask if there's any update regarding this.

Thank you for your ongoing work on the project, and I hope you all had a pleasant Easter.

Best regards.

**Comment by Visiraptor** [comment]
Dear Developer(s),

following up on my message from April about a new stable release. I haven't heard back. I noticed a "Pre-release" last month ("Build 211") with a long changelog, but no new stable version. It's been over six months since my last inquiry and the latest stable release is almost two years old... any update on a stable release schedule? How many more "Pre-releases" should we expect before a new stable release?

Thanks for your work on the project. I hope you all had a pleasant Christmas, and I wish you an early Happy New Year.

Best regards.

PS: Certainly I (and possibly others) have been very patient... it's been over a year now since my initial message and I am still waiting. Should we give up? If a stable release isn't planned, please let us know, so users can decide whether to continue waiting (or switch to alternatives etc.).

**Comment by arekf** [comment]
I have sent several builds to App Store this year, none of the iOS got accepted, they are rejected by Apple App Store review on different basis, from app being "spam" to "too similar to other apps". The same tvOS build got accepted 🤷

I will continue pushing new builds to App Store but it's outside my control if it's accepted so don't ask when or if it will be updated.

**Reply by Visiraptor** [reply]
Thanks for the update. I appreciate the (semi-)transparency.

A few quick requests that would help users understand (more):

- Do you know (or can you guess) why the app was accepted up to version 1.5.1 for iOS but not for newer releases? What changed? I know there have been many changes over the last year(s), so no pressure if this is hard to pinpoint.
- Could you summarize which App Store rejections were blocking the iOS release (specific rejection reasons or guideline numbers for the latest "Pre-release")? [Maybe someone from the community could then try to help/fix also.]
- If possible, can you share whether you'll pursue appeal or code/metadata changes to address those rejections? [Yes, we cannot "control" what Apple AppStore accepts/rejects, but might try to solve the issues/mentions, which might guide to acceptance at some point. It is btw. "strange" to me that it gets accepted for tvOS but not iOS, but I know nothing about the differences/specifics etc. besides the obvious TV vs. non-TV, so there is that.]
- Would you consider publishing an official status note in the repo (README or Releases) stating current tvOS availability, iOS status (planned/paused/...), and why there hasn't been a stable release since January 2024? [Because then more people would be aware about it (without going to the "discussions"-section) and maybe would like to get involved as potential contributors. Thanks again that you clarified it meanwhile in here. Better later than never 😅.]

It's great that we're all on the same side trying to get the app accepted by Apple.

Thanks again for the work and for pushing builds — that context is helpful.

**Reply by NovaAndrom3da** [reply]
Hi! Is there any chance you'd be able to throw it on TestPilot for the meantime?

**Comment by bbsixzz** [comment]
Would you please consider releasing on https://altstore.io for us Europeans?
